composer.json

```diff
@@ -85,6 +85,9 @@
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
+        ],
+        "build-parameters": [
```

Comment:
I created it for being able to have a hook only generating configuration based on the environment, which I call [before actually starting wallabag](https://github.com/mathbruyen/home/blob/master/wallabag/Dockerfile#L19). I trigger the install fetching dependencies while generating the container but I only _patch_ the configuration from the environment.

That's maybe not the best way to do, I've never used composer before.